lang/es-es.js

```diff
@@ -0,0 +1,23 @@
+/* eslint quotes: 0 */
+
+export default {
+	"appliedFilters": "Filtros aplicados:",
+	"noActiveFilters": "No hay filtros activos",
```

Comment:
Wondering if this specific term is backwards between `es` and `es-es`... this was the langterm from `es-mx`, and we rewrite from `es-mx` to `es`.  BUT we don't want to put the `es-es` term for `es`... so maybe both should be `Sin filtros activos`?

Reply:
Yeah I agree -- it's hard to reconcile but before `es-es` would have resolved to `es` so it makes sense I think for them to both be the same. Updated.